tests/integration/targets/ios_vrf_global/vars/main.yaml

```diff
@@ -0,0 +1,111 @@
+---
+merged:
+  before: {}
```

Comment:
we should assert more before dicts! Helps in keeping track of changes that might effect the module and also helps in troubleshooting tests.